plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestS3WrongRegionPicked.java

```diff
@@ -23,16 +23,17 @@
 
 public class TestS3WrongRegionPicked
 {
-    @Test
+    @Test(invocationCount = 100)
     public void testS3WrongRegionSelection()
             throws Exception
     {
-        try (HiveMinioDataLake dataLake = new HiveMinioDataLake("test-bucket").start();
+        String bucketName = "test-bucket" + randomTableSuffix();
+        try (HiveMinioDataLake dataLake = new HiveMinioDataLake(bucketName).start();
                 QueryRunner queryRunner = S3HiveQueryRunner.builder(dataLake)
```

Comment:
Maybe add simple setMaxRetries to TestingThriftHiveMetastoreBuilder->thriftConfig would do the trick? Right now there is only 1 retry with 20 seconds of timeout.

Reply:
I actually don't think retrying this will help.
The original error is that there are no env variables present. Retrying will not fix that probably.

Reply:
Yes, it is a local macOS thing.